python/social_bot/models/agent_cfg.json

```diff
@@ -41,8 +41,37 @@
             "youbot::wheel_joint_fl",
             "youbot::wheel_joint_fr"],
         "camera_sensor": "",
-        "use_pid": false,
-        "max_sim_step_time": 0.0025
+        "use_pid": true,
+        "pid_type": [
+            "position",
```

Comment:
use positional PID for robot arm and velocity PID for wheels. This is the same as most real robot cases.

Reply:
Does json support comment? If possible, it's better to comment in json.

Reply:
Sadly strict JSON does not allow comments... everything is data.  Comments can be added as unused data in a way like {“value”: “the value”, “comment”: “the comment”}